File: daq/slc/dqm/src/Graph1C.cc
#include "daq/slc/dqm/Graph1C.h"

#include "daq/slc/dqm/CharArray.h"

using namespace Belle2;

Graph1C::Graph1C() : Graph1()
{
  m_data = new CharArray(0);
}

Graph1C::Graph1C(const Graph1C& h) : Graph1(h)
{
  m_data = new CharArray(*h.m_data);
}

Graph1C::Graph1C(const std::string& name, const std::string& title,
                 int nbinx, double xmin, double xmax, double ymin, double ymax)
  : Graph1(name, title, nbinx, xmin, xmax, ymin, ymax)
{
  m_data = new CharArray(nbinx * 2);
}

Graph1C::~Graph1C() throw() {}

std::string Graph1C::getDataType() const throw()
{
  return std::string("g1C");
}

File: daq/slc/extra/javalib/b2daq/src/b2daq/dqm/core/Graph1C.java
package b2daq.dqm.core;


public class Graph1C extends Graph1 {

	public Graph1C() {
		super();
		data = new CharArray(0);
	}
	public Graph1C(String name, String title, int nbinx, double xmin, double xmax,
			double ymin, double ymax) {
		super(name, title, nbinx, xmin, xmax, ymin, ymax);
		data = new CharArray(nbinx*2);
	}
	public Graph1C(Graph1C h) {
		super();
		data = new CharArray(0);
	}
	public String getDataType() {
		return "g1C";
	}

}
